Themis/Views/Assessment/CodeEditor/CodeView.swift

```diff
@@ -12,17 +12,19 @@ struct CodeFile: Identifiable, Hashable {
 
 // integrates the UITextView of runestone in SwiftUI
 struct CodeView: UIViewControllerRepresentable {
-    @ObservedObject var model: AssessmentViewModel
+    @ObservedObject var vm: CodeEditorViewModel
+    @ObservedObject var selectedFile: Node
```

Comment:
What's the reason for separating the `selectedFile` into its own variable instead of directly accessing it from `AssessmentViewModel`?

Reply:
No reason. Should have already been removed. I removed now.